install/error-handling.sh

```diff
@@ -47,22 +48,19 @@ send_event() {
   # Add header to specify the event type of envelope to be sent
   echo '{"type":"event"}' >>$envelope_file_path
 
-  # Next we construct the meat of the event payload, which we build up
+  # First we construct the meat of the event payload, which we build up
```

Comment:
I think this was correct as is, this isn't the first part of the payload we create.

Reply:
fair enough, I was just bothered by two `Next` in a row, changed the other one to `Then` haha